sql/src/main/java/io/crate/metadata/settings/session/SessionSetting.java

```diff
@@ -61,6 +65,10 @@ public void apply(Row parameters, List<Expression> expressions, SessionContext s
     }
 
     public String getValue(SessionContext sessionContext) {
```

Comment:
Would it generally be possible to only have a `getValue(SessionSettings)` ? so that we don't have to have 2 getters for each session setting